app/controllers/questions_controller.rb

```diff
@@ -90,8 +90,7 @@ def recently_commented
       .where(status: 1)
     @pagy, @questions = pagy(filter_questions_by_tag(@questions, params[:tagnames])
       .joins(:comments)
-      .order('comments.timestamp DESC')
-      .group('node.nid'), items: 24)
```

Comment:
I have removed the `group` function from here for two reasons:

1. The `@questions` we are filtering on are already being grouped by `node.nid`.
2. The MySQL issue that is coming up was that ordering was happening on a variable on which grouping is not happening.

I believe that because of this, the behaviour of the code should not change, and we should get the same results.

Reply:
I think we are doing this to order it by the comment timestamps, but because nodes can have multiple comments, we had been seeing them appear more than once, so we re-added the group to ensure each node appears only once. But if you can confirm for sure that the duplication isn't happening, I like a simpler query!

Reply:
Hmm...on the data that I have I'm unable to see any duplication, but I think it is safer to keep it in that way then.